src/GDGUkraine/model.py

```diff
@@ -223,6 +234,16 @@ def __init__(self, **kwargs):
     participants = relationship("EventParticipant", backref="events")
     host_gdg = relationship("Place", backref="events")
 
+    def is_registration_open(self):
```

Comment:
@webknjaz: method name is already expressive enough to make it clear that it does not change data. So that is just a matter of taste. I prefer method.